packages/astroport-governance/src/voting_escrow.rs

```diff
@@ -65,47 +72,65 @@ pub enum ExecuteMsg {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum Cw20HookMsg {
+    /// Create a vxASTRO position and lock xASTRO for `time` amount of time
     CreateLock { time: u64 },
+    /// Deposit xASTRO in another user's vxASTRO position
     DepositFor { user: String },
+    /// Add more xASTRO to your vxASTRO position
     ExtendLockAmount {},
 }
 
 /// ## Description
-/// This structure describes the query messages of the contract.
+/// This structure describes the query messages available in the contract.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum QueryMsg {
+    /// Return the user's vxASTRO balance
     Balance { address: String },
+    /// Fetch the vxASTRO token information
     TokenInfo {},
+    /// Fetch vxASTRO's marketing information
     MarketingInfo {},
+    /// Download the vxASTRO logo
     DownloadLogo {},
+    /// Return the current total amount of vxASTRO
     TotalVotingPower {},
+    /// Return the total amount of vxASTRO at some point in the past
     TotalVotingPowerAt { time: u64 },
+    /// Return the user's vxASTRO balance
     UserVotingPower { user: String },
+    /// Return the user's vxASTRO balance at some point in the past
     UserVotingPowerAt { user: String, time: u64 },
+    /// Return a user's vxASTRO position details
     LockInfo { user: String },
+    /// Return the  vxASTRO contract configuration
     Config {},
 }
 
 /// ## Description
-/// This structure describes voting power response.
+/// This structure is used to return a user's amount of vxASTRO.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct VotingPowerResponse {
+    /// The vxASTRO balance
     pub voting_power: Uint128,
 }
 
 /// ## Description
-/// This structure describes lock information response.
+/// This structure is used to return the lock information for a vxASTRO position.
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 pub struct LockInfoResponse {
+    /// The amount of xASTRO locked in the position
     pub amount: Uint128,
+    /// How fast the lock position voting power decays
```

Comment:
This is initial boost which was applied during lock creation or when lock's amount was increased. Voting power decay (slope) is not exposed in queries.